app/assets/stylesheets/modules/panel_list.css.scss

```diff
@@ -80,12 +80,18 @@ $panel-list-thumbnail-size: $panel-list-item-height;
   float: left;
   position:relative;
   margin-right: .5em;
+  width: $panel-list-thumbnail-size + 10px;
+  height: $panel-list-thumbnail-size + 10px;
+  background-color: #222;
+  border: 5px solid #323232;
 }
 
 .panel-list-thumbnail {
   height: $panel-list-thumbnail-size;
```

Comment:
Change line 17 of granule.coffee from:

    "#{scalerUrl}/granules/#{@id}?h=#{h}&w=#{w}"

to:

    "http://placehold.it/30x65"

Which is an image taller than it is wide. The code looks correct.  Now change it to:

    "http://placehold.it/65x30"

Which is an image wider than it is tall. The code stretches it into a square. It should be centered and maintain aspect ratio.